bundles/org.openhab.binding.vektiva/src/main/java/org/openhab/binding/vektiva/internal/config/VektivaSmarwiConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.vektiva.internal.config;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link VektivaSmarwiConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Ondrej Pecta - Initial contribution
+ */
+@NonNullByDefault
```

Comment:
I think this is wrong. `refreshInterval` and `useWebSockets` are not initialized. Either add `@Nullable` to these fields or remove the class annotation.

Reply:
I have added default value to the  initialization